src/services/upstream/upstream.js

```diff
@@ -120,7 +119,7 @@ export default class UpstreamService {
     const valueEth = valueBn.div(new BigNumber(10).pow(18)).toString(10);
     const transaction = {
       to,
-      data: normalizeHexValue(data),
```

Comment:
@borjantrajanoski Is there any reason to remove `normalizeHexValue` call? I think it should be there.

Reply:
@seland It was removed on purpose as ```normalizeHexValue``` was being called one more time [here](https://github.com/Bit-Nation/BITNATION-Pangea-mobile/blob/d23fbb55049f174de69aad7c3297e83c88789e88/src/services/ethereum/CustomSigner.js#L113)

This was causing the data to be prefixed by an additional `0x` which was causing the data to get malformed.